management/azure_mgmt_network/lib/generated/azure_mgmt_network/available_private_access_services.rb

```diff
@@ -0,0 +1,224 @@
+# encoding: utf-8
```

Comment:
Not an issue with a generation. since they are new files, my git checkout . command did not delete them and I did not notice it. My bad. Removed it now